.rubocop_base.yml

```diff
@@ -17,7 +17,7 @@ Layout/ExtraSpacing:
 Layout/SpaceBeforeFirstArg:
   Enabled: false
 Layout/SpaceInsideHashLiteralBraces:
-  Enabled: false
+  EnforcedStyle: no_space
```

Comment:
Is the default no_space?  If so, you can just remove this cop from the file altogether.

Reply:
No, the default is `space`